SocialCareCaseViewerApi/V1/Boundary/Requests/CreateAllocationRequest.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+{
+    public class CreateAllocationRequest
```

Comment:
Ideally we should have test coverage for boundary objects as well. Both individual properties and validations for each should be tested. This would ensure any changes to boundary objects are done safely.

Reply:
Tests will be added when the boundary object is more clearly defined